en/docs/consume/discover-apis/marketplace-assistant.md

```diff
@@ -53,10 +49,94 @@ To enable the Marketplace Assistant and populate the vector database, the API Ma
 Explore the features and functionalities of the Marketplace Assistant to streamline your API management experience further.
+
+## FAQ
+
+**1. Why is this an experimental feature?**
+
+The Marketplace Assistant is labeled as an experimental feature because it represents innovative technology that is still under development. AI technology is rapidly evolving with significant enhancements in large language models and the introduction of increasingly efficient chips, and we are integrating the latest advancements to enhance our capabilities. Rolling out new features as experimental underscores our commitment to innovation, and as we continue to refine them, we are focused on enhancing their performance and reliability based on user input and ongoing development efforts.
+
+**2. What are the usage limits?**
+
+During the trial period, users are subject to the following usage limits:
+
+- Access to up to 1000 APIs to the vectorDB.
+- Limited chat functionality based on the number of tokens used by the language model.
+
+**3. Why did I lose my chat history after I logged out?**
+
+Your chat history will be lost after logging out or after a period of inactivity of 1 hour, as per the design of the application. For security and privacy reasons ensure that your conversations are not accessible to other users. Additionally, you have the option to reset your chat history to address any potential privacy concerns.
+
+**4. Why does the chat not have the context of the initial interactions?**
+
+The language model operates by considering only the last 5 interactions to respond. Therefore, if your conversation extends beyond 5 interactions, the model may not retain the earlier parts of the chat
+
+**5. Why can't I list a large number of APIs as recommendations?**
+
+This is because the assistant is designed to identify the most suitable APIs rather than listing a large number of options. To enhance the recommendation process, we limit the number of APIs provided to the language model as context. This ensures that the assistant suggests more relevant APIs instead of overwhelming the user with every possible option.
+
+**6. I'm receiving the message "I am not aware of such API" despite the API being listed on the developer portal.**
+
+There are a few potential reasons for this
+
+1.  Assistant uses only public APIs to provide answers.
+
+    The marketplace assistant is only using public APIs to answer questions. If there is an API, not a public API, the assistant wouldn't be aware of it.
+
+2. You may have configured the marketplace assistant after publishing those APIs.
+
+    APIs are only published to the vector database if the Al features are enabled and the token is configured. If you published the API before configuring the assistant, it might not be in the database, and the assistant wouldn't be aware of it. Re-publishing the API and trying again could resolve this. If you have numerous APIs, there's an option to upload them using the WSO2 API Controller
```

Comment:
```suggestion
    APIs are only published to the vector database if the AI features are enabled and the token is configured. If you published the API before configuring the assistant, it might not be in the database, and the assistant wouldn't be aware of it. Re-publishing the API and trying again could resolve this. If you have numerous APIs, there's an option to upload them using the WSO2 API Controller
```